Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM golang:1.3
 
-ADD . /go/src/github.com/docker/swarm
+COPY . /go/src/github.com/docker/swarm
 WORKDIR /go/src/github.com/docker/swarm
 
-RUN go get github.com/tools/godep
-RUN godep go install
+ENV GOPATH $GOPATH:/go/src/github.com/docker/swarm/Godeps/_workspace
+RUN go install -v
 
 EXPOSE 2375
-ENTRYPOINT ["swarm"]
+CMD ["swarm"]
```

Comment:
I'm just going to give my 2 cents here, because isn't that what open source is all about ;)

I think it should be `ENTRYPOINT` because we should use the best practices for our Dockerfiles. So:
```console
$ docker run -d swarm swarm --version
```
is kinda weird when it should be:
```console
$ docker run -d swarm --flag my --cluser is --awesomer than --everyone elses
```

Also if you are concerned with people not knowing the difference between `ENTRYPOINT` and `CMD` this would be a great example to show if you did something like:
```
ENTRYPOINT ["swarm"]
CMD ["--help"]
```